ferrischat_webserver/src/entrypoint.rs

```diff
@@ -110,17 +110,12 @@ pub async fn entrypoint() {
             // GET    guilds/{guild_id}/members/{member_id}
             .route(
                 expand_version!("guilds/{guild_id}/members/{member_id}"),
```

Comment:
You need to remove the entire route() fn call for it to actually be removed. Right now it'll just compile error.

Reply:
AKA delete lines 105-109